contracts/ArtifactApplication.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity 0.5.8;
```

Comment:
Not sure if we should use ^0.5.12 instead to say we want any version above 0.5.12. We seemed to go against this in an earlier commit https://github.com/artistic-cryptids/ARRtistry/commit/07ca6860b8da0a7593618c0f5c48796787ebac7e